ras/lshw.py

```diff
@@ -123,9 +126,9 @@ def test_lshw_verification(self):
             self.fail("lshw failed to show correct mac address")
 
         # verify network
-        if self.active_interface\
-                not in self.run_cmd_out("lshw -class network"):
-            self.fail("lshw failed to show correct active network interface")
+        if 'ppc64le' not in distro.detect().arch:
+            if self.active_interface not in self.run_cmd_out("lshw -class network"):
```

Comment:
i think it is applicable to ppc64le arch as well

Reply:
agree